### DISCUSSION TITLE
Get BASE64 v3.x

### DISCUSSION THREAD

**Discussion by TiagoBehenck** [react-native-image-picker/react-native-image-picker#1575] [General]
How i get a base64 from a image in version 3.x. In the objetct reponse has a key `base64` but this returns undefined. 

I use `react-native 0.63`

**Comment by jorgeferreira008** [comment]
I have  this doubt to.

**Comment by Johan-dutoit** [comment]
includeBase64 | OK | OK | If true, creates base64 string of the image (Avoid using on large image files due to performance)
-- | -- | -- | --
